config/main.py

```diff
@@ -1983,8 +1983,23 @@ def load_minigraph(db, no_service_restart, traffic_shift_away, override_config,
                         fg='magenta')
             raise click.Abort()
 
-        # Dependency check golden config json
+        cfgdb_clients = db.cfgdb_clients
+        # Pick the host config in case of multi-asic
+        config_db = cfgdb_clients[DEFAULT_NAMESPACE]
+
+        cm = None
+        try:
+            cm = ConfigMgmt(configdb=config_db)
+            cm.validateConfigData()
+        except Exception as ex:
+            log.log_warning("The existing config fails YANG validation. Alerting: {}".format(ex))
```

Comment:
Do we need exit for this exception? or is there real case that validateConfigData() gets failed and validate_config_by_cm() gets succeed?

Reply:
validateConfigData will validate current running config. validate_config_by_cm here is to validate golden config.
We cannot assume the running config is already yang validated. We only make sure it is yang validted after load_minigraph+override by golden config.

Reply:
Changed to sonic_yang to avoid CONFIG_MGMT init failure